File: src/JavaScript/Stopwatch/Stopwatch.js
const fs = require('fs');
const read = fs.readFileSync(0, 'utf8').split('\n');

let current = 0;
function input() {
    return read[current++];
}

const n = input();
if (n % 2 != 0) {
    console.log('still running');
} else {
    let sum = 0;
    for (let i = 0; i < n; i++) {
        let line = +input();
        if (i % 2 == 0) {
            sum -= line;
        } else {
            sum += line;
        }
    }
    console.log(sum);
}

File: src/Python 3/Stopwatch/Stopwatch.py
n = int(input())
if not n % 2 == 0:
    print('still running')
else:
    sec = 0
    for i in range(n):
        if i % 2 == 0:
            sec -= int(input())
        else:
            sec += int(input())
    print(sec)